### ISSUE TITLE
Search criteria not applied in tasks dashboard after using browser back button

### ISSUE THREAD

**Issue by mhwilder** [opencv/cvat#695]
The workflow we're using for reviewing annotations involves applying a search to show tasks that need review (the search query is "status:validation"). When we do this it works fine and we can click into a task to review it. Once the review is complete, the status of the task is marked as "completed", the work is saved, and the browser back button is used to get to the main task list. The problem is at this point, the task list is the full list without the search query applied (though the search query still shows in the box). It is possible to have the search query reapplied when the back button is used so the user doesn't have to manually click the search icon each time they come back to the task list?

We're experience this in Chrome version 76.

**Comment by nmanovic** [comment]
@mhwilder , we will investigate and try to support the scenario if it is possible.

**Comment by bsekachev** [comment]
@mhwilder 
You can open a job in a new task (press mouse-wheel).

**Comment by mhwilder** [comment]
@bsekachev Thanks for the suggestion. I don't have a mouse wheel, but I assume you're just suggesting opening the job in a new tab so that the main task list (with the search filter) remains in the original tab. I tried this and it doesn't quite solve the problem because I'm changing the status of the job being reviewed (and the status is what is being used as the search criterion for the main task list). The ideal functionality would be for the task to be removed from the task list when I go back to it because the status no longer matches the search criterion.

I realize that this might be a somewhat specific use case. However, I do think the process of reviewing annotations should be supported with an intuitive, smooth process. I'm open to hearing other suggestions for reviewer workflows that you guys have. Thanks!
